src/app/api/review/write/hook.ts

```diff
@@ -0,0 +1,11 @@
+import { useQuery } from "@tanstack/react-query";
+import { getPurpose } from "@app/api/review/write";
+
+export const usePurposeGet = () => {
+  return useQuery({
+    queryKey: ["purpose"],
+    queryFn: () => {
+      return getPurpose();
+    },
```

Comment:
p3) 무언가 내부적으로 추가 작업을 해주어야하는 게 아니라면, 바로 queryFn 넣어도 좋을 것 같아요.
```suggestion
    queryFn: getPurpose,
```

Reply:
넵! 반영하겠습니다!